packages/client/hmi-client/src/components/widgets/tera-input-field.vue

```diff
@@ -25,14 +27,33 @@ const emit = defineEmits(['update:modelValue']);
 const inputField = ref<HTMLInputElement | null>(null);
 const attrs = useAttrs();
 
-const textAlign = computed(() => (attrs.type === 'number' ? 'right' : 'left'));
+const textAlign = computed(() =>
+	attrs.type === 'number' || attrs.type === 'sci' ? 'right' : 'left'
+);
 const focusInput = () => {
 	inputField.value?.focus();
 };
 
+const getValue = () =>
+	attrs.type === 'sci' ? maskToNistNumber(props.modelValue) : props.modelValue;
+
 const updateValue = (event: Event) => {
-	const value = (event.target as HTMLInputElement).value;
-	emit('update:modelValue', value);
+	const target = event.target as HTMLInputElement;
+	const value = target.value;
+
+	if (attrs.type === 'sci') {
+		// Mask the value before emitting the update
+		const maskedValue = maskToNistNumber(value);
+
+		emit('update:modelValue', maskedValue);
```

Comment:
As briefly discussed, "mask" has a specific meaning in computer-science (usually involving bitwise operations) and it doesn't look like it applies here, if we are just formatting I think calling formatXYZ will be better for code understanding.

Reply:
I looked a little further this morning, the term is `input masking` and is used to mask user input into certain formats https://designsystem.digital.gov/components/input-mask/.